src/services/contentfulClient.js

```diff
@@ -10,22 +11,25 @@ function initClient (space, accessToken, preview) {
     host: preview ? 'preview.contentful.com' : 'cdn.contentful.com'
   })
   return client.getSpace()
-  .then(() => {
-    authorized = true
-    return authorized
-  })
+    .then((space) => {
+      authorized = true
+      currentSpace = space
+      return space
+    })
 }
 
 function getClient () {
   return authorized && client
 }
 
+function getCurrentSpaceName () {
+  let currentSpaceName = (currentSpace && currentSpace.name) ? currentSpace.name : ''
+  return currentSpaceName
+}
+
 function resetClient () {
+  window.sessionStorage.clear()
```

Comment:
I saw that already when I took some parts for the gallery. Where is something set to the storage. :)